src/amp-story-player/amp-story-entry-point/amp-story-entry-point-impl.js

```diff
@@ -84,6 +113,170 @@ export class AmpStoryEntryPoint {
     shadowRoot.appendChild(this.rootEl_);
   }
 
+  /** @private */
+  initializeCarousel_() {
+    const style = this.element_.getAttribute('entry-point-style');
+    const styleClass = !Object.values(STYLE_TYPES).includes(style)
+      ? STYLE_CLASSES['rectangle']
+      : STYLE_CLASSES[style];
+
+    this.element_.classList.add(styleClass);
+
+    const cardsContainer = this.doc_.createElement('div');
+    cardsContainer.classList.add('entry-points');
+    this.rootEl_.append(cardsContainer);
+
+    for (let i = 0; i < this.stories_.length; i++) {
+      const story = this.stories_[i];
+      const src = story.getAttribute('data-poster-portrait-src');
+
+      const cardContainer = this.doc_.createElement('div');
+      cardContainer.classList.add('entry-point-card-container');
+      cardsContainer.append(cardContainer);
+
+      if (styleClass !== 'circular-entry-point') {
```

Comment:
This should be changed to check for which styles do not display the logo if other styles get added.